sdk/app_cpu1/common/drv/timing_manager.h

```diff
@@ -9,23 +9,30 @@
 #include <stdbool.h>
 #include <stdint.h>
 
-#define INTC_INTERRUPT_ID_0      (61) // IRQ_F2P[0:0]
-#define ISR0_PRIORITY            (0x08)
-#define ISR_RISING_EDGE          (0x3)
+#define TM_INTERRUPT_ID          (61) // IRQ_F2P[0:0]
+#define TM_ISR_PRIORITY          (0x08)
+#define ISR_RISING_EDGE          (3)
 #define DEFAULT_SENSOR_ENABLE    (0x0000)
-#define DEFAULT_PWM_RATIO        (0x1)
+#define DEFAULT_PWM_RATIO        (10)
 #define LOWER_16_MASK            (0x0000FFFF)
 #define NUM_SENSORS              (10)
 #define TIMING_MANAGER_BASE_ADDR (XPAR_AMDC_TIMING_MANAGER_0_S00_AXI_BASEADDR)
 #define UPPER_16_SHIFT           (16)
 
+// Expected times (us) for all the sensors based on default values
+#define ADC_DEFAULT_TIME        (0.82)
+#define ENCODER_DEFAULT_TIME    (0.005)
+#define AMDS_DEFAULT_TIME       (11.8)
+#define EDDY_DEFAULT_TIME       (4.11)
+#define MAX_DEFAULT_SENSOR_TIME (AMDS_DEFAULT_TIME)
```

Comment:
Can you prepend `TM_` to these defines to enforce a "name space"? You have this on the first 2, but the rest are not name spaced and might clash with other files where this is included.